cfme/utils/appliance/db.py

```diff
@@ -197,17 +193,20 @@ def loosen_pgssl(self, with_ssl=False):
         status, out = client.run_command("systemctl restart {scl}-postgresql".format(scl=scl))
         return status
 
-    def enable_internal(self, region=0, key_address=None, db_password=None, ssh_password=None):
+    def enable_internal(self, region=0, key_address=None, db_password=None, ssh_password=None,
+                        db_disk=None):
         """Enables internal database
 
         Args:
             region: Region number of the CFME appliance.
             key_address: Address of CFME appliance where key can be fetched.
+            db_disk: Path of the db disk for --dbdisk appliance_console_cli. If not specified it
+                     tries to load it from the appliance.
 
         Note:
             If key_address is None, a new encryption key is generated for the appliance.
         """
-        self.logger.info('Enabling internal DB (region {}) on {}.'.format(region, self.address))
+        # self.logger.info('Enabling internal DB (region {}) on {}.'.format(region, self.address))
```

Comment:
I commented this out, but I think we can drop it below the `self.address` assignment to retain this log info.